bodhi/server/util.py

```diff
@@ -684,3 +686,101 @@ def sort_severity(value):
     }
 
     return value_map.get(value, 99)
+
+
+def get_critpath_pkgs_from_pdc(branch):
+    """
+    Searches PDC for critical path packages based on the specified branch
+    :param branch: string representing the branch
+    :return: a list of critical path packages
+    """
+    pdc_api_url = '{}/rest_api/v1/component-branches/'.format(
+        config.get('pdc_url', 'https://pdc.fedoraproject.org/').rstrip('/'))
+    query_args = urllib.urlencode({
+        'active': 'true',
+        'critical_path': 'true',
+        'name': branch,
+        'page_size': 100,
+        'type': 'rpm'
```

Comment:
I just checked with Ralph, and he pointed that it does support RPMs, containers, and modules. I removed the type limitation here. So now it will filter just by the collection/branch like PkgDB does.